src/travis2docker/templates/entrypoint_deployv.sh

```diff
@@ -58,7 +58,10 @@ def start_psql():
 def list_modules(modules_path):
     main_repo_path = os.environ["MAIN_REPO_FULL_PATH"]
     list_modules = []
+    excluded_modules = set(os.environ["EXCLUDE"].split(","))
```

Comment:
- use `get`, because the variable might not be defined
- Use strip on every module, to support spaces after commas
- Discard empty strings

Reply:
Thanks for the check @luisg123v. I have fixed the errors. I will now fix the python 3.6 containers and it should be ready to go.